Dockerfile

```diff
@@ -141,7 +142,8 @@ RUN source /etc/default/evm && \
 RUN source /etc/default/evm && \
     cd ${SUI_ROOT} && \
     yarn install && \
-    yarn run build
+    yarn run build && \
+    rm -rvf node_modules
```

Comment:
That was one of the "other things we could consider" things that I was going to add to this as well, which was removing `node` entirely after this step.  From what I can tell, we are only using `node` in MIQ to build assets, and not as part of the running application, so we only need it in place to "compile the app", but not at runtime.

That said, for this to have any effect on the resulting container, we would have to basically combine all of the `RUN` steps together because of how Docker's layer system works, otherwise it would just be dead weight.   That assumes we are just using the Dockerfile here to build the image that is pushed to docker hub, but if we are doing an export of the resulting filesystem that removes the layer data, then this is less of a problem.

I will some up my ideas for improvement in a separate comment, and we can decide if they are worth pursuing and moving to a formal issue/PR.